src/Framework/AspNetCoreAnalyzers/test/WebApplicationBuilder/DisallowConfigureHostLoggingTest.cs

```diff
@@ -46,117 +47,129 @@ public static void Main (string[] args)
 }
 public class Startup { }
 ";
-        //act
-        var diagnostics = await Runner.GetDiagnosticsAsync(source);
+
         //assert
-        Assert.Empty(diagnostics);
+        await VerifyCS.VerifyCodeFixAsync(source, source);
```

Comment:
What are the two arguments to VerifyCodeFixAsync? I haven't used it before. Looks odd to pass the same thing twice.

Reply:
`VerifyCodeFixAsync` and `VerifyCS` are APIs provided by the new analyzer testing libraries that we've been replatting onto.

Since we're only asserting that not diagnositcs are thrown it would be more appropriate to use `VerifyCS.VerifyAnalyzerAsync` with an empty expected diagnostics property here.